doc/yay.8

```diff
@@ -532,6 +552,16 @@ Shows statistics for installed packages and system health.
 pacman -Qmq | grep -Ee '-(cvs|svn|git|hg|bzr|darcs)$' | yay -S --needed -
 pacaur-like devel check.
 
+.SH ENVIRONMENT VARIABLES
+.TP
+.B AURDEST
+Can be set to configure the builddir.
+
+.TP
+.B VISUAL EDITOR
```

Comment:
```suggestion
.B VISUAL 
.B EDITOR
```
Disregard if it doesn't work well visually

Reply:
It does not work.

![image](https://user-images.githubusercontent.com/16593899/67214139-2002f280-f417-11e9-82fe-895cdd336c9a.png)


Maybe I should have but a comma though thinking about it.

Reply:
or a `/` , because I know there will be some who will try and set a `VISUAL EDITOR` env flag.
It will work though `EXPORT VISUAL EDITOR=vim` :sweat_smile: